README.md

```diff
@@ -13,51 +13,57 @@ The followings are required for building Kilda controller:
  - Gradle 6.7+
  - Maven 3.3.9+
  - JDK8
- - Python 2.7+
  - Python 3.5+
  - Docker 19.03.3+
  - Docker Compose 1.20.0+
  - GNU Make 4.1+
  - Open vSwitch 2.9+
  
-For running virtual environment you additionally need linux kernel 4.18+ for OVS meters support
-
-On Ubuntu 18.04, you can install those dependencies like this:
 
+#### Dependency installation on Ubuntu 18.04 
+For running virtual environment you additionally need linux kernel 4.18+ for OVS meters support. A virtual environment 
+in that sentence means docker instance with open-vswitch in it. We use that for functional testing.
+```
+sudo apt install maven make openjdk-8-jdk openvswitch-switch python3-pip linux-generic-hwe-18.04
```

Comment:
why do we need linux-generic-hwe-18.04?

Reply:
because of meters support in the kernel from 4.18+ and ubuntu 18.04 has only linux-image-4.15.0-161-generic